src/App/App.js

```diff
@@ -14,15 +15,41 @@ class App extends Component {
   }
 
   componentDidMount() {
-    getArtists().then(data => console.log(data))
-    console.log('hello');
+    this.getRandomArtist()
   }
 
 
+  getRandomArtist() {
+    let artistAmount = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 12, 13, 14, 15]
+    let artistId = artistAmount.map((num) => {
+      let radomNumber = Math.random() * (300 - num) + num;
```

Comment:
This is such a clever way to set up getting random artists!! On line 25 + 26, you might wanna rename radomNumber to randomNumber to eliminate confusion.